lib/glific_web/resolvers/flows.ex

```diff
@@ -74,4 +76,30 @@ defmodule GlificWeb.Resolvers.Flows do
       {:ok, %{success: true}}
     end
   end
+
+  @doc false
+  @spec start_contact_flow(Absinthe.Resolution.t(), %{id: integer, contact_id: integer}, %{
+          context: map()
+        }) ::
+          {:ok, any} | {:error, any}
+  def start_contact_flow(_, %{id: id, contact_id: contact_id}, _) do
+    with {:ok, flow} <- Repo.fetch(Flow, id),
```

Comment:
Why do we need to fetch the flow here? can't you just pass this to context and get_Cached_flow will load it.

Reply:
get_cached_flow doesn't have a check for incorrect data, we need to fix it to make it work